docs/notebooks/ppg/ppg_analysis.ipynb

```diff
@@ -48,22 +44,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Shape of the original data: (64775, 2) (72947, 4)\n",
+      "Shape of the overlapping segments: (64775, 2) (64361, 4)\n"
+     ]
+    }
+   ],
```

Comment:
Je hebt hier en daar nog execution counts, wat op zich niet een probleem is omdat je misschien juist deze output wilt laten zien aan een gebruiker zonder dat een gebruiker de code hoeft te draaien. Even een check of dat je intentie was.

Reply:
Ehm uiteindelijk wel, maar niet voor nu eig dus excuses